pkg/executor/aggregate/agg_hash_executor.go

```diff
@@ -432,24 +470,26 @@ func (e *HashAggExec) prepare4ParallelExec(ctx context.Context) {
 		partialWallTimePtr = &e.stats.PartialWallTime
 		finalWallTimePtr = &e.stats.FinalWallTime
 	}
+
 	partialWorkerWaitGroup := &sync.WaitGroup{}
 	partialWorkerWaitGroup.Add(len(e.partialWorkers))
 	partialStart := time.Now()
 	for i := range e.partialWorkers {
 		go e.partialWorkers[i].run(e.Ctx(), partialWorkerWaitGroup, len(e.finalWorkers))
 	}
-	go func() {
```

Comment:
why remove this?

Reply:
> why remove this?

In new implementation, final worker could work only when partial workers are finished so we should be blocked here.